### PR TITLE
I18n update to develop

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

let's see if this helps address https://github.com/learningequality/kolibri/issues/6605

### Reviewer guidance

…

### References

…

----

### Contributor Checklist

PR process:

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/locale/es_ES/LC_MESSAGES/kolibri.core.default_frontend-messages.json
Comment (radinamatic) [inline kolibri/locale/es_ES/LC_MESSAGES/kolibri.core.default_frontend-messages.json:44]:
```suggestion
  "CommonCoreStrings.coachLabel": "Tutor",
```
I'm confused why is this being changed to `Tutor` again, when I explicitly set to be `Tutorizar` back in December? Same below  for `CommonCoachStrings.coachLabel`

Comment (radinamatic) [reply kolibri/locale/es_ES/LC_MESSAGES/kolibri.core.default_frontend-messages.json:44]: Weird regression, my change on Crowdin was probably overwritten by an intermediate upload. 

@indirectlylit I'm going to approve so we can test if the in-context server can be made to work with this , and I'll have to correct these 2 strings again on Crowdin.

**Review by radinamatic — APPROVED** [review]
Yes, let's do this! :+1:

**Comment by lyw07** [comment]
should I deploy this to the translations server once this gets merged? or wait until we have a beta for 0.14.0? thank you!

**Comment by radinamatic** [comment]
We should not wait for 0.14. The idea is to see if this fixes the error we're experiencing on the in-context server.

**Comment by indirectlylit** [comment]
@lyw07 merged and ready to test on the translation server

**Pull request merged**
